src/node.cc

```diff
@@ -2844,6 +2845,11 @@ void SetupProcessObject(Environment* env,
     // after LoadEnvironment() has run.
   }
 
+  // --track-heap-objects
+  if (track_heap_objects) {
+    READONLY_PROPERTY(process, "trackHeapObjects", True(env->isolate()));
+  }
```

Comment:
Good to know.  @bmeck Can you remove the `.trackHeapObjects` property (and `.traceSyncIO` in a separate commit if you want)?